main-app/app/scripts/gameboard/gameboard-controller/gameBoard.js

```diff
@@ -0,0 +1,21 @@
+angular.module('Tombola.MyModule')
+.controller('MyController', ['$scope','$q', 'Server', 'ServerTurn','FreshGame', 'CoreData','BothHumans', function ($scope,$q, server, serverTurn, freshGame, coreData, bothHumans) {
```

Comment:
In this you generally need to separate 
a) The initital current player
b) When to swap the player during gameplay.